tests/ImageSharp.Benchmarks/Codecs/Jpeg/DecodeJpeg_ImageSpecific.cs

```diff
@@ -115,5 +115,24 @@ public CoreSize JpegImageSharp()
         //                                 |                                             |            |            |           |        |          |           |          |          |             |
         //  'Decode Jpeg - System.Drawing' |       Jpg/issues/issue750-exif-tranform.jpg |  95.192 ms |  3.1762 ms | 0.1795 ms |   1.00 |     0.00 | 1750.0000 |        - |        - |  5492.63 KB |
         //      'Decode Jpeg - ImageSharp' |       Jpg/issues/issue750-exif-tranform.jpg | 230.158 ms | 48.8128 ms | 2.7580 ms |   2.42 |     0.02 |  312.5000 | 312.5000 | 312.5000 | 58834.66 KB |
+
```

Comment:
It's a separate library `Quamotion.TurboJpegWrapper`. I was reticent to do so as it uninstalled some underlying 64 bit library and installed a 32 bit version. I can't even find what dependency it would have been. There seem to be no issues though so I'll add it.

Reply:
Decided against including the library for now.